homeassistant/components/netatmo/camera.py

```diff
@@ -146,12 +154,19 @@ def handle_event(self, event: dict) -> None:
             data["home_id"] == self.home.entity_id
             and data["camera_id"] == self.device.entity_id
         ):
-            if data[WEBHOOK_PUSH_TYPE] in ("NACamera-off", "NACamera-disconnection"):
+            if data[WEBHOOK_PUSH_TYPE] in (
+                "NACamera-off",
+                "NOCamera-off",
+                "NACamera-disconnection",
+                "NOCamera-disconnection",
+            ):
                 self._attr_is_streaming = False
                 self._monitoring = False
             elif data[WEBHOOK_PUSH_TYPE] in (
                 "NACamera-on",
+                "NOCamera-on",
                 WEBHOOK_NACAMERA_CONNECTION,
```

Comment:
It's very inconsistent this has a constant and the other ones are hard coded strings.
We don't need to fix that in this PR though.